pai-management/k8sPaiLibrary/maintainlib/common.py

```diff
@@ -216,9 +224,15 @@ def ssh_shell_paramiko_with_result(host_config, commandline):
         if port_validation(host_config['sshport']) == False:
             return (None, None)
         port = int(host_config['sshport'])
+    key_filename = None
+    if 'key-filename' in host_config and host_config['key-filename'] is not None:
+        if os.path.isfile(str(host_config['key-filename'])):
+            key_filename = str(host_config['key-filename'])
+        else:
+            logger.warn("The key file: {0} specified doesn't exist".format(host_config['key-filename']))
```

Comment:
If the admin configure the field ```ssh-key-filename```  or ```key_filename```, but paictl failed to find it. Is it better to exit from script?

Reply:
I am not sure the principle of PAI development. If user provide both keyfile and password, but keyfile paht is wrong, maybe we should try password as well?

Reply:
@hao1939  What's your opinion?